sonar-text-plugin/src/main/java/org/sonar/plugins/secrets/api/SpecificationLoader.java

```diff
@@ -63,17 +56,32 @@ private static Map<String, List<Rule>> initialize(String specificationLocation,
       try {
         specification = loadSpecification(specificationLocation, specificationFileName);
       } catch (DeserializationException | SchemaValidationException e) {
-        LOG.error(String.format("Could not load specification from file: %s", specificationFileName), e);
+        LOG.error(String.format("%s: Could not load specification from file: %s", e.getClass().getSimpleName(), specificationFileName));
```

Comment:
Will it work like this?
```suggestion
        LOG.error("{}: Could not load specification from file: {}", e.getClass().getSimpleName(), specificationFileName);
```